### PR TITLE
Fix 11112 - Rename {srwx,flags,perms} to perm. (-21 LOC)

### PR DESCRIPTION
**Pull request by radare**
Unify R_IO, R_BIN, R_BP, .. into R_PERM_* using 1 letter syntax

### CODE REVIEW

**Comment by radare** [comment]
Probably missed to update some JSONs to be consistent too. so i expect tests to break

**Comment by radare** [comment]
Also we have to update some help messages and commands like `omf`.. which should be `omp` f was flags p is perm

**Comment by ret2libc** [comment]
> Also we have to update some help messages and commands like omf.. which should be omp f was flags p is perm

`omp` is already for `priority` :(

File: libr/io/desc.c [hunk 1]
```diff
@@ -243,17 +243,18 @@ R_API bool r_io_desc_is_chardevice(RIODesc *desc) {
 	return desc->plugin->is_chardevice (desc);
 }
 
-
+#if 0
```
Comment (ret2libc) [inline libr/io/desc.c]: please just remove

**Comment by radare** [comment]
i know its about priority, and i dont plan to change this in this pr because it probably requires more tests and checkings, and this PR is big enough already.

**Comment by radare** [comment]
```
[0K
[31m[XX][39m db/cmd/cmd_i [33miSSj[39m  53

$ r2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Q -i /tmp/tmp-157285dnD2zhfMjVO.tmp ../bins/elf/analysis/x86-helloworld-gcc
iSSj

[35m-[{"name":"PHDR","size":256,"vsize":256,"flags":"-r-x","paddr":52,"vaddr":134512692},{"name":"INTERP","size":19,"vsize":19,"flags":"-r--","paddr":308,"vaddr":134512948},{"name":"LOAD0","size":1436,"vsize":1436,"flags":"-r-x","paddr":0,"vaddr":134512640},{"name":"LOAD1","size":280,"vsize":284,"flags":"-rw-","paddr":1436,"vaddr":134518172},{"name":"DYNAMIC","size":232,"vsize":232,"flags":"-rw-","paddr":1448,"vaddr":134518184},{"name":"NOTE","size":68,"vsize":68,"flags":"-r--","paddr":328,"vaddr":134512968},{"name":"GNU_EH_FRAME","size":44,"vsize":44,"flags":"-r--","paddr":1216,"vaddr":134513856},{"name":"GNU_STACK","size":0,"vsize":0,"flags":"-rw-","paddr":0,"vaddr":0},{"name":"ehdr","size":52,"vsize":52,"flags":"-rw-","paddr":0,"vaddr":134512640}][39m
[35m[39m[32m+[{"name":"PHDR","size":256,"vsize":256,"perm":"-r-x","paddr":52,"vaddr":134512692},{"name":"INTERP","size":19,"vsize":19,"perm":"-r--","paddr":308,"vaddr":134512948},{"name":"LOAD0","size":1436,"vsize":1436,"perm":"-r-x","paddr":0,"vaddr":134512640},{"name":"LOAD1","size":280,"vsize":284,"perm":"-rw-","paddr":1436,"vaddr":134518172},{"name":"DYNAMIC","size":232,"vsize":232,"perm":"-rw-","paddr":1448,"vaddr":134518184},{"name":"NOTE","size":68,"vsize":68,"perm":"-r--","paddr":328,"vaddr":134512968},{"name":"GNU_EH_FRAME","size":44,"vsize":44,"perm":"-r--","paddr":1216,"vaddr":134513856},{"name":"GNU_STACK","size":0,"vsize":0,"perm":"-rw-","paddr":0,"vaddr":0},{"name":"ehdr","size":52,"vsize":52,"perm":"-rw-","paddr":0,"vaddr":134512640}][39m
[32m[39m
EXPECT=<<RUN
```

**Comment by ret2libc** [comment]
Could you start a PR in r2r and put the link here?

**Comment by radare** [comment]
of course, but there are still things to be fixed in here, not in r2r, so i prefer to reduce the noise before fixing tests

**Comment by ret2libc** [comment]
Ok. I thought you were already fixing tests.

**Comment by radare** [comment]
https://github.com/radare/radare2-regressions/pull/1497

**Comment by radare** [comment]
Should be green but travis doesnt seems to grab my branch :{{

**Pull request merged**